scripts/gcs/flask_gcs.py

```diff
@@ -176,7 +176,7 @@ def __init__(self):
         self.targets_to_submit = self.manager.list()
 
         self.interop_client = self.manager.list()
-        self.interop_client.append(SUASSystem.InteropClientConverter())
+        #self.interop_client.append(SUASSystem.InteropClientConverter())
```

Comment:
Please uncomment this. Behavior shouldn't accidentally change when we merge the code into the master branch.